src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/content/main.js

```diff
@@ -17,7 +17,12 @@ define(function() {
                 url: '/admin/content-navigations?alias=account'
             },
             toolbar: {
-                template: 'default'
+                buttons: [
+                    'saveWithOptions',
+                    {'settings': {
+                        dropdownItems: ['delete']
+                    }}
```

Comment:
Is `saveWithOptions` something like a template, to which we add the settings button? Or what should this actually do?